airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py

```diff
@@ -132,5 +132,5 @@ def create_slack_message(self) -> str:
             assert self.report is not None, "Report should be set when state is successful"
             message += f"⏲️ Run duration: {format_duration(self.report.run_duration)}\n"
         if self.state is ContextState.FAILURE:
-            message += "\ncc. <!subteam^S03BQLNTFNC>"
+            message += "\ncc. <!subteam^S077R8636CV>"
```

Comment:
Just curious where this came from.

Reply:
That's from Slack client! Tap on the team mention, three dots > copy team id. Hopefully this works 🙃